### DISCUSSION TITLE
How to Load Data from Inactive player

### DISCUSSION THREAD

**Discussion by axiades** [plan-player-analytics/Plan#2465] [Q&A]
Hello,

i want to ask how to load All Player Data from the server into PLAN.

The server is already 1 month old and PLAN are only loading the data from Player who was online since i installed PLAN.
I am using PaperMC 1.18.2 Latest Build

Thanks to all who try to help me.

**Comment by AuroraLS3** [comment]
Hi, you can try `plan import offline`, but it may not work because of https://github.com/plan-player-analytics/Plan/issues/1882

**Reply by axiades** [reply]
okay-- dont works

Crap
